pages/synths.tsx

```diff
@@ -11,46 +11,79 @@ import exchangeInfoQuery, { Fees, Rates } from 'src/queries/exchangeInfo/useExch
 import getSNXJS from 'src/lib/snxjs';
 import useMarketClosed from 'src/hooks/useMarketClosed';
 import { SynthStatus } from 'src/sections/synths/SynthCard';
-import { NetworkIdByName, Synth } from '@synthetixio/contracts-interface';
+import { Synth } from '@synthetixio/contracts-interface';
 
-export async function getStaticProps() {
-	const tokenListResponse = await axios.get<TokenListResponse>('https://synths.snx.eth.link');
-	const tokenList = {
-		tokens: tokenListResponse.data.tokens,
-		tokensMap: keyBy(tokenListResponse.data.tokens, 'symbol'),
-		symbols: tokenListResponse.data.tokens.map((token) => token.symbol),
-	};
-	const exchangeInfo = await exchangeInfoQuery();
-	const synths = getSNXJS({ useOvm: false, networkId: NetworkIdByName.mainnet }).synths;
+const getSynthStatus = async (useOvm: boolean) => {
+	const synths = getSNXJS({ useOvm }).synths;
 	const dictionarySynthStatus: Record<string, SynthStatus> = {};
 	const promises = synths.map(async (synth) => {
-		const marketClosed = await useMarketClosed(synth.name);
+		const marketClosed = await useMarketClosed(synth.name, useOvm);
 		if (marketClosed.isMarketClosed) {
 			dictionarySynthStatus[synth.name] = SynthStatus.PAUSED;
 		} else {
 			dictionarySynthStatus[synth.name] = SynthStatus.LIVE;
 		}
 	});
 	await Promise.all(promises);
+
+	return {
+		synths,
+		dictionarySynthStatus,
+	};
+};
+
+export async function getStaticProps() {
+	const [
+		tokenListResponse,
+		{ exchangeInfoL1, exchangeInfoL2 },
+		{ synths: synthsL1, dictionarySynthStatus: dictionarySynthStatusL1 },
+		{ synths: synthsL2, dictionarySynthStatus: dictionarySynthStatusL2 },
+	] = await Promise.all([
+		axios.get<TokenListResponse>('https://synths.snx.eth.link'),
```

Comment:
I guess we can get rid of that? cause we get every information form chain? The link only returns L1 synths anyways, right?

Reply:
well we are getting the "logoURI" property for SynthCard. I don't think that data can is onChain.

Reply:
can't we do something like that here:
```
	<StyledSynthImage
				src={`https://raw.githubusercontent.com/Synthetixio/synthetix-assets/v2.0.13/synths/${name}.svg`}
			/>
```
https://github.com/Synthetixio/synthetix-website/blob/staging/src/components/FutureSynthCard/index.tsx#L16
all the svgs are stored in that repo